### PR TITLE
Fixed white bar at smaller widths

### LINKED ISSUE #53: Fix smallest screen width (320px) bug 🐛
Image

### PR DESCRIPTION
**Pull request by Lunchii**
Closes #53 
## Overview
The text from the footer component made the screen stretch due to the big font size.
Changed the text so that it changes for mobile and is now viewable

### CODE REVIEW

File: app/home/page.tsx [hunk 1]
```diff
@@ -8,6 +8,10 @@ function Home() {
         <div className="w-1/2 bg-slate-700 float-left text-center">
             Home...
         </div>
+<<<<<<< Updated upstream
+=======
+
+>>>>>>> Stashed changes
```
Comment (Cheekane) [inline app/home/page.tsx:14]: Fix this.

**Review by Cheekane — CHANGES_REQUESTED** [review]
@Lunchii Get rid of useless images.

**Review by jerpyk — COMMENTED** [review]
👍
Screenshot 2024-12-28 at 6 53 25 PM

**Review by ZestyScrubz — APPROVED** [review]
Good Job, but it still breaks on very small screen, but that's no problem

**Review by Cheekane — APPROVED** [review]
🦖

**Pull request merged**